### ISSUE TITLE
Math equation

### ISSUE THREAD

**Issue by zihaosino** [sbarex/QLMarkdown#110]
# I wander this will support mathjax render
just like adding below code in html can render math equation
```html
<script type="text/x-mathjax-config">MathJax.Hub.Config({tex2jax: {inlineMath:[['$','$']]}});</script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=default' async></script>
```

**Comment by yig** [comment]
I would like an option for math support. GitHub markdown [now supports math](https://github.blog/2022-05-19-math-support-in-markdown/). It would be nice to be able to preview that in QLMarkdown.

**Comment by WillawZ** [comment]
This may be a temporary solution
[Glance.zip](https://github.com/sbarex/QLMarkdown/files/10311381/Glance.zip)
I added some code to the other qlmarkdown project glance, original project here:
https://github.com/samuelmeuli/glance

Do note that this is a very crude solution, it should render mathematical equations for most of the times but some equations it will refuse to render.

Also sorry if this is a bad place to post for such a thing, I've never really used github before.

If you find better solutions for the problem do tell me, I may just have worked a week for nothing.

**Comment by thanhqtran** [comment]
It seems to me that [iPreview](https://apps.apple.com/us/app/ipreview-powerful-quick-look/id1519213509?mt=12) is the only app that supports Math typeset. 

I hope QLMarkdown can add this feature soon.

**Comment by sbarex** [comment]
Implemented on release 1.0.16 (41).
